.jshintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This is essentially our usual style.  It's in jshintrc because I followed the hot-shot setup and am just using "npm test" instead of Grunt.